src/components/form/currency-select.js

```diff
@@ -0,0 +1,106 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { graphql, useStaticQuery } from 'gatsby'
+import { Flex } from '../containers'
+import { QueryImage, Header } from '../elements'
+
+const Item = styled(Flex)`
+    width: 90px;
+    pointer-events: all;
+    cursor: pointer;
+    font-size: 12px;
+    padding: 0.8rem 1.6rem;
+    transition: background 0.25s;
+
+    &:hover {
+        background: rgba(245, 247, 250, 0.64);
+    }
+`
+
+const StyledContentFlex = styled(Flex)``
+
+const CurrencySelect = ({ setFieldValue, current_select }) => {
+    const query = graphql`
+        query {
+            usd: file(relativePath: { eq: "currencies/usd.png" }) {
+                ...fadeIn
+            }
+            eur: file(relativePath: { eq: "currencies/eur.png" }) {
+                ...fadeIn
+            }
+            gbp: file(relativePath: { eq: "currencies/gbp.png" }) {
+                ...fadeIn
+            }
+            aud: file(relativePath: { eq: "currencies/aud.png" }) {
+                ...fadeIn
```

Comment:
These would be better served as SVGs to maintain it's clarify considering it's small size.

Reply:
Updated with regular` img` tag